gst/tensor_reposrc/tensor_reposrc.c

```diff
@@ -175,6 +177,18 @@ gst_tensor_reposrc_getcaps (GstBaseSrc * src, GstCaps * filter)
       cap = gst_caps_new_any ();
   }
 
+  st = gst_caps_get_structure (cap, 0);
+
+  name = gst_structure_get_name (st);
+
+  if (!g_str_equal (name, "other/tensor")
+      && !g_str_equal (name, "other/tensors")) {
+    GST_ELEMENT_ERROR (GST_ELEMENT (self), STREAM, WRONG_TYPE,
+        ("Only Tensor/Tensors MIME are supported for now"), (NULL));
+  }
+
+  gst_tensors_config_from_structure (&self->config, st);
```

Comment:
Why do this here?

Reply:
To get the size of tensors. We could do this without call this.